bundles/org.openhab.binding.vesync/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -109,6 +113,8 @@
 			<channel id="humiditySetpoint" typeId="deviceConfigTargetHumidity"/>
 			<channel id="warmEnabled" typeId="warmModeEnabled"/>
 			<channel id="warmLevel" typeId="warmLevel"/>
+			<channel id="schedulesCount" typeId="deviceAFConfigAutoScheduleCountType"/>
```

Comment:
airHumidifier thing also needs tthe thingTypeVersion property, otherwise the upgrade instructions won't work